### PR TITLE
Move Sean to Emeritus

### PR DESCRIPTION
**Pull request by AMoo-Miki**
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] All tests pass
  - [ ] `yarn lint`
  - [ ] `yarn test-unit`
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -16,3 +15,10 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Josh Romero              | [joshuarrrr](https://github.com/joshuarrrr)       | Amazon      |
 | Andrey Myssak            | [andreymyssak](https://github.com/andreymyssak)   | Contributor |
 | Sergey Myssak            | [SergeyMyssak](https://github.com/SergeyMyssak)   | Contributor |
+
+
+## Emeritus
+
+| Maintainer    | GitHub ID                                 | Affiliation |
+| ------------- | ----------------------------------------- | ----------- |
```
Comment (joshuarrrr) [inline MAINTAINERS.md:23]: nit - for consistency with top table
```suggestion
| Maintainer               | GitHub ID                                         | Affiliation |
| ------------------------ | ------------------------------------------------- | ----------- |
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -49,6 +49,7 @@
 - Bump `@types/react` and `csstype` ([#1105](https://github.com/opensearch-project/oui/pull/1105))
 - Add `scripts` folder to lint-es script ([#1143](https://github.com/opensearch-project/oui/pull/1143))
 - Update deprecated Babel plugins ([#1155](https://github.com/opensearch-project/oui/pull/1155))
+- Add @seanneumann to the emeritus maintainer ([#1188](https://github.com/opensearch-project/oui/pull/1188))
```
Comment (joshuarrrr) [inline CHANGELOG.md:52]:
```suggestion
- Move @seanneumann to emeritus maintainer ([#1188](https://github.com/opensearch-project/oui/pull/1188))
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -16,3 +15,10 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
...
+| Sean Neumann             | [seanneumann](https://github.com/seanneumann)     | Amazon      |
```
Comment (joshuarrrr) [inline MAINTAINERS.md:24]: Should this affiliation be updated?

**Review by joshuarrrr — APPROVED** [review]

File: MAINTAINERS.md (hunk 3 above)
Comment (AMoo-Miki) [reply MAINTAINERS.md:24]: Donno. I followed OSD's.

**Review by BSFishy — APPROVED** [review]

**Review by seanneumann — COMMENTED** [review]
w00t!

**Pull request merged**